sdk/tvOS/OoyalaTVSkinSDK/OOTVOptionsCollectionView.m

```diff
@@ -0,0 +1,34 @@
+//
+//  OOTVOptionsCollectionView.m
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/21/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "OOTVOptionsCollectionView.h"
+
+@implementation OOTVOptionsCollectionView
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    return self;
+}
+
+- (BOOL)canBecomeFocused {
+    return NO;
+}
+
+- (BOOL)shouldUpdateFocusInContext:(UIFocusUpdateContext *)context {
+    return YES;
+}
+
+- (NSArray<id<UIFocusEnvironment>> *)preferredFocusEnvironments {
+    if (self.subviews != nil){
+        return self.subviews;
+    }
+    return self;
```

Comment:
You try to return `self` but it doesn't conform to the `UIFocusEnvironment`. There's a warning when you do this in Xcode. Maybe you don't need to do anything, but just make sure everything is setup accordingly.